internal/jimm/juju/model_status_parser.go

```diff
@@ -20,6 +23,10 @@ import (
 	"github.com/canonical/jimm/v3/pkg/api/params"
 )
 
+var (
+	jqQueryDeadline = time.Second * 5
```

Comment:
What about rather making this a parameter on the struct then you don't need to make this a global.

Reply:
Yeah I'll make it a param, I've no idea what would be best in terms of timing it out.